charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 11.0.0
+version: 12.0.0
```

Comment:
imho this is currently a breaking change for anybody who uses this currently.

i would change it in a way that the default values you've set are only used if the ports are not set.